Create a communication channel

I've just started a brand new Rust project with Cargo, replaced `lib.rs` by the [chat example](https://github.com/libp2p/rust-libp2p/blob/9102266d7031c024e2fbce48198478edb0e40728/examples/chat.rs), added those dependencies:

```
[dependencies]
libp2p = { git = "https://github.com/libp2p/rust-libp2p" }
futures = "0.1"
env_logger = "0.6.0"
tokio = "0.1"
```

Attempted to run it with `cargo +nightly run`, got this error:

```
   Compiling foo v0.1.0 (/Users/v/vic/dev/bitlog)
error[E0433]: failed to resolve: use of undeclared type or module `tokio_stdin_stdout`
   --> src/main.rs:103:17
    |
103 |     let stdin = tokio_stdin_stdout::stdin(0);
    |                 ^^^^^^^^^^^^^^^^^^ use of undeclared type or module `tokio_stdin_stdout`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0433`.
error: Could not compile `foo`.
```

Did I forget to do anything?

Also: is there any place where I can ask newbie libp2p questions?

--- a/README.md
+++ b/README.md
@@ -11,15 +11,15 @@ cannot guarantee that this is the case considering the lack of a precise libp2p 
 
 ## Documentation
 
-This repository includes a façade crate named `libp2p`, which reexports the rest of the repository.
+How to use the library?
 
-For documentation, you are encouraged to clone this repository or add `libp2p` as a dependency in
-your Cargo.toml and run `cargo doc`.
+- Main documentation: https://docs.rs/libp2p
 
-```toml
-[dependencies]
-libp2p = "0.2.2"
-```
+Where to ask questions?
+
+- In the Rust section of https://discuss.libp2p.io.
+- In the #libp2p IRC channel on freenode.
+- By opening an issue in this repository.
 
 ## Notable users
 

